### DISCUSSION TITLE
How to Pass different dataTypes while Passing Input Parameters ,if possible avaialable datatypes supported and how we needed to pass different string to specific formats

### DISCUSSION THREAD

**Discussion by NDayakarChowdary** [dbosoft/YaNco#268] [Q&A]
![image](https://github.com/dbosoft/YaNco/assets/119785305/66bc9235-d617-4b74-9806-51eb3e2eb585)
![image](https://github.com/dbosoft/YaNco/assets/119785305/e90e3fc8-20a8-45be-a101-1d3a557b3cfa)
 needed to convert because always passing in the form of String

**Comment by fw2568** [comment]
I don't know what your question is. Please try to write a clear title, description of your question and use text instead of screenshots for code.
